index.html

```diff
@@ -291,6 +314,30 @@ <h3> Normative Specifications </h3>
                 Associated <a href="https://lists.w3.org/Archives/Member/member-cfe/2018May/0003.html">Call for Exclusion</a> on 10 May 2018, ending on 9 July
                 2018<br>
                 Produced under <a href="https://www.w3.org/2016/05/timed-text-charter.html">2016 Working Group Charter</a></dd>
+                <br/>
+              <dt id="adpt" class="spec">TTML Profile for Audio Description</dt>
+              <dd><p>This specification defines a profile of 
+                [<cite><a class="bibref" href="https://www.w3.org/TR/ttml-imsc1.1/#bib-TTML2">TTML2</a></cite>]
```

Comment:
Citation is wrong, needs to point to TTML generic URL `https://www.w3.org/TR/ttml/`, and omit TTML version number.